.github/workflows/docker-image-build.yml

```diff
@@ -0,0 +1,56 @@
+name: Docker Build Image
+
+on:
+  release:
+      types:
+          - "released"
+
+jobs:
+  main:
+    runs-on: ubuntu-18.04
```

Comment:
Can we start using 20.04 yet?

Reply:
I believe we can. I'll modify the Dockerfile to use 20.04 as the base image.